lib/videojs-playlist-ui.js

```diff
@@ -66,6 +66,7 @@ const createThumbnail = function(thumbnail) {
     // simple thumbnails
     let img = document.createElement('img');
     img.src = thumbnail;
+    img.alt = '';
```

Comment:
what about using the description of the video or the name of the video instead? it should be available somewhere, hopefully.